// Copyright 2023 The Chromium Authors
// Use of this source code is governed by a BSD-style license that can be
// found in the LICENSE file.

syntax = "proto2";

option java_multiple_files = true;
option java_package = "org.chromium.components.password_manager.core.browser.proto";

option optimize_for = LITE_RUNTIME;

package password_manager;

// Additional password information to be displayed in the UI.
message PasswordInfo {
  // Display name for websites/apps.
  optional string display_name = 1;

  // Favicon url.
  optional string icon_url = 2;
}